packages/cli/src/reporters/abstract-list.ts

```diff
@@ -101,6 +103,44 @@ export default abstract class AbstractListReporter implements Reporter {
     printLn(chalk.red('Unable to run checks: ') + err.message)
   }
 
+  onStreamLogs (check: any, sequenceId: SequenceId, logs: Array<{ timestamp: number, message: string }> | undefined) {
+    const checkFile = this.checkFilesMap!.get(check.getSourceFile?.())!.get(sequenceId)!
+    const logList = logs || []
+    if (!checkFile.logs) {
+      checkFile.logs = []
+    }
+
+    // Display the check title if this is the first time we're streaming logs for this check
+    const isFirstLogBatch = checkFile.logs.length === 0
+    if (isFirstLogBatch) {
+      // For Playwright tests, we need to create a better display name
+      const displayCheck = {
+        ...check,
+        sourceFile: check.getSourceFile?.() || 'Playwright Test',
+        name: check.name || check.logicalId || 'Unknown Test',
+      }
+      printLn(formatCheckTitle(CheckStatus.RUNNING, displayCheck, { includeSourceFile: true }))
+    }
+
+    // Format and display each log with proper indentation and timestamp handling
+    logList.forEach(logEntry => {
+      // Format timestamp from Unix timestamp to HH:mm:ss.SSS format
+      const timestamp = DateTime.fromMillis(logEntry.timestamp).toFormat('HH:mm:ss.SSS')
+      const formattedLog = `[${timestamp}] ${logEntry.message}`
+
+      // Handle logs that contain newlines by splitting and indenting each line
+      const logLines = formattedLog.split('\n')
+      logLines.forEach((line, index) => {
+        if (line.trim()) { // Only print non-empty lines
+          printLn(indentString(line, 4), 1, 0) // Always add 1 newline after each line
```

Comment:
I'm a bit confused by this. Won't a multi-line log line end up looking like this:

```
> console.log(require('indent-string')('[22:03:11.739] foo\nbar\nqux', 4))
    [22:03:11.739] foo
    bar
    qux
```

And even for single-line log lines, the line will still be indented with 4 spaces.

Is this as intended?